api/api.py

```diff
@@ -205,3 +205,22 @@ def listdir(directory: str, recursive: bool = False) -> List[str]:
   """Returns all the files at the top level of the provided directory. If ``recursive`` is set to True, returns all files that are inside of the provided directory, recursively."""
   pass
 
+def k8s_kind(kind: str, *, image_json_path: Union[str, List[str]]):
+  """Tells Tilt about a k8s kind. Primarily intended for defining where your CRD specifies image names.
+
+  (Note the `*` in the signature means `image_json_path` must be passed as a keyword, e.g., `image_json_path="{.spec.image}"`)
+
+  Example ::
+
+    # Fission has a CRD named "Environment"
```

Comment:
Can you make this formatted with ```? (not sure if you can)

Reply:
That's [taken care of](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) by the `::`

![image](https://user-images.githubusercontent.com/7453991/53838338-cff81e00-3f62-11e9-9954-cce4d3a66325.png)